src/year2/components/Timeline.tsx

```diff
@@ -0,0 +1,157 @@
+import React from "react";
+
+import classes from "./Timeline.module.scss";
+
+type MemoryCommon = {
+  title: string;
+  member: "elira" | "finana" | "pomu";
+  date: Date;
+  submitterName: string;
+  submitterSocialUrl?: string;
+};
+
+type YouTubeMemory = MemoryCommon & {
+  type: "youtube";
+  videoId: string;
+};
+
+type TwitterMemory = MemoryCommon & {
+  type: "twitter";
+  tweetUrl: string;
```

Comment:
Of note: if you use publish.twitter.com it will give you some HTML. To avoid us having to manually go there to generate this HTML for every tweet, I've instead opted to take tweets as just the tweet URL and reconstruct a skeleton of the twitter-generated `blockquote` (see `TwitterMemory` below) with all of the tweet-specific content removed.